amr-wind/overset/TiogaInterface.cpp

```diff
@@ -291,37 +386,68 @@ void TiogaInterface::amr_to_tioga_mesh()
         }
     }
 
+
+
+    // Synchronize data on host/device
+    m_amr_data->level.copy_to_device();
+    m_amr_data->mpi_rank.copy_to_device();
+    m_amr_data->local_id.copy_to_device();
+    m_amr_data->ilow.copy_to_device();
+    m_amr_data->ihigh.copy_to_device();
+    m_amr_data->dims.copy_to_device();
+    m_amr_data->xlo.copy_to_device();
+    m_amr_data->dx.copy_to_device();
+    m_amr_data->global_idmap.copy_to_device();
+
+}
+void TiogaInterface::amr_to_tioga_iblank()
+{
+    BL_PROFILE("amr-wind::TiogaInterface::amr_to_tioga_iblank");
+    auto& mesh = m_sim.mesh();
+    const int nlevels = mesh.finestLevel() + 1;
+
     // Reset local patch counter
-    ilp = 0;
+    int ilp = 0;
     auto& ibcell = m_sim.repo().get_int_field("iblank_cell");
     auto& ibnode = m_sim.repo().get_int_field("iblank_node");
+
+
+    // Create standard vector of integer pointers here
+    auto& ad = *m_amr_data;
+    amrex::Vector<int *> tmpdataPtr(ad.iblank_cell.size());
+    amrex::Vector<int *> tmpdataPtrn(ad.iblank_node.size());
     for (int lev = 0; lev < nlevels; ++lev) {
-        auto& ad = *m_amr_data;
+        //auto& ad = *m_amr_data;
```

Comment:
remove this

Reply:
I must have missed it. Thanks for pointing it out.